topo_processor/cli/main_test.py

```diff
@@ -0,0 +1,54 @@
+import json
+import os
+import shutil
+from tempfile import gettempdir
+
+import pytest
+
+from topo_processor.metadata import DataType, create_collection
+
+from .main import upload_to_local_disk
+
+
+@pytest.fixture(autouse=True)
+def teardown():
+    yield
+    shutil.rmtree(os.path.join(gettempdir(), "C8054"))
+
+
+@pytest.mark.asyncio
+async def test_local_save_paths():
+
+    source = os.path.join(os.getcwd(), "test_data", "tiffs", "C8054")
+    datatype = "imagery.historic"
+    target = gettempdir()
+
+    collection = await create_collection(source, DataType(datatype))
+    await upload_to_local_disk(collection, target)
+
+    assert os.path.isfile(os.path.join(target, "C8054", "29659.json"))
+    assert os.path.isfile(os.path.join(target, "C8054", "29659.tiff"))
+    assert os.path.isfile(os.path.join(target, "C8054", "collection.json"))
+
+
+@pytest.mark.asyncio
+async def test_item_contents():
+
+    source = os.path.join(os.getcwd(), "test_data", "tiffs", "C8054")
+    datatype = "imagery.historic"
+    target = gettempdir()
```

Comment:
should use a actual temp folder not the `/tmp` i think there is a `mktempdir` or similar on `tempfile`